packages/components/src/logos/bloomberg-logo/index.tsx

```diff
@@ -1,5 +1,12 @@
 const BloombergLogo = ( props: React.SVGProps< SVGSVGElement > ) => (
-	<svg xmlns="http://www.w3.org/2000/svg" width={ 69 } height={ 27 } fill="none" { ...props }>
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
+		width={ 69 }
+		height={ 27 }
+		viewBox="0 0 69 27"
```

Comment:
Added `viewBox` properties to the Bloomberg, CNN, and Condenast logos to allow them to be scalable.